learn/ai_powered_search/getting_started_with_ai_search.mdx

```diff
@@ -17,72 +17,154 @@ This tutorial will walk you through configuring AI-powered search in your Meilis
+AI-powered search is an experimental feature and is disabled by default. You must manually activate it either via the Meilisearch Cloud UI, or with the experimental features endpoint.
 
-### Meilisearch Cloud projects
+<Capsule intent="tip" title="Meilisearch Cloud AI-powered search waitlist">
+To use AI-powered search with Meilisearch Cloud, you must first enter the waitlist. You will not be able to activate vector search until your sign-up has been approved.
+</Capsule>
 
-If using Meilisearch Cloud, navigate to your project overview and find "Experimental features". Then check the "AI-powered search" box.
+### Meilisearch Cloud UI
 
-![A section of the project overview interface titled "Experimental features". The image shows a few options, including "Vector store".](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vector-search/01-cloud-vector-store.png)
+Navigate to your project overview and find "Experimental features". Then click on the "AI-powered search" box.
 
-<Capsule intent="note" title="Meilisearch Cloud AI-powered search waitlist">
-To ensure proper scaling of Meilisearch Cloud's latest AI-powered search offering, you must enter the waitlist before activating vector search. You will not be able to activate vector search in the Cloud interface or via the `/experimental-features` route until your sign-up has been approved.
-</Capsule>
+![A section of the project overview interface titled "Experimental features". The image shows a few options, including "Vector store".](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vector-search/01-cloud-vector-store.png)
 
-### Self-hosted instances
+### Experimental features endpoint
 
-Use [the `/experimental-features` route](/reference/api/experimental_features?utm_campaign=vector-search&utm_source=docs&utm_medium=vector-search-guide) to activate vector search during runtime:
+Use [the `/experimental-features` route](/reference/api/experimental_features) to activate vector search during runtime:
 
 ```sh
 curl \
-  -X PATCH 'http://localhost:7700/experimental-features/' \
+  -X PATCH 'http://MEILISEARCH_URL/experimental-features/' \
   -H 'Content-Type: application/json'  \
   --data-binary '{
     "vectorStore": true
   }'
 ```
 
-## Generate vector embeddings with OpenAI
+## Generate embeddings with OpenAI
+
+In this step, you will configure an OpenAI embedder. Meilisearch uses **embedders** to translate documents into **embeddings**, which are mathematical representations of a document's meaning and context.
+
+Open a blank file in your text editor. You will only use this file to build your embedder one step at a time, so there's no need to save it if you plan to finish the tutorial in one sitting.
+
+### Choose an embedder name
+
+In your blank file, create your `embedder` object:
+
+```json
+{
+  "kitchenware-openai": {}
```

Comment:
I understand that you chose a name that is not directly `openai` so that users don't confuse the embedder name with the `source`. I'm not sure I like prefixing the index name though. Ideally we'd find something else 🤔. If we don't, we can leave it as-is